proto/storage/alert.proto

```diff
@@ -14,21 +14,21 @@ option java_package = "io.stackrox.proto.storage";
 
 message Alert {
   message Deployment {
-    string id = 1 [(gogoproto.moretags) = 'search:"Deployment ID,store,hidden" sql:"index=hash,type(uuid)"'];
-    string name = 2 [(gogoproto.moretags) = 'search:"Deployment,store"'];
+    string id = 1 []; // @gotags: search:"Deployment ID,store,hidden" sql:"index=hash,type(uuid)"
+    string name = 2 []; // @gotags: search:"Deployment,store"
     string type = 4;
     string namespace = 5; // This field has to be duplicated in Alert for scope management and search.
     string namespace_id = 16; // This field has to be duplicated in Alert for scope management and search.
     map<string, string> labels = 7 [(gogoproto.moretags) = 'sensorhash:"ignore"'];
```

Comment:
Why wasn't this one converted?

Reply:
Fixed as maps are a separated node https://github.com/janisz/buf/commit/9fd7c64fa75582d90c49b39d5a51e4017b10edeb